### PR TITLE
Add statement to readme

### PR DESCRIPTION
**Pull request by han-so1omon**
Add statement about purpose and contents of icon.community site

Signed-off-by: Eric Solomon <eric@icon.foundation>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # icon.community – The Official ICON Community Website
 
-The icon.community website is the main hub to direct members of the ICON ecosystem to available resources.
+The icon.community website is the main hub to direct members of the ICON ecosystem to available resources. This site includes blogs, links to other media, and general information about the ICON community.
```
Comment (DavidFBD) [inline README.md:3]: This is too generic. Please re-write to tickle the imagination.

**Review by DavidFBD — CHANGES_REQUESTED** [review]
Please see prior comment in read-me file.

File: README.md (hunk 1 above)
Comment (han-so1omon) [reply README.md:3]: I fixed it

**Review by DavidFBD — APPROVED** [review]
This works for me.

**Pull request merged**